recipe/build.sh

```diff
@@ -1,18 +1,17 @@
 #!/bin/bash
-# Get an updated config.sub and config.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.* .
 
-export XERCESCROOT=${PREFIX}
-export XALANCROOT=${SRC_DIR}
+# Force C++ 17
+perl -pi -e "s#CMAKE_CXX_STANDARD\s+\d+#CMAKE_CXX_STANDARD 17#g" CMakeLists.txt 
 
-if [[ ${target_platform} == osx-64 ]]; then
-    platform=macosx
-else
-    platform=linux
-    export CXXCPP=${CPP}
-fi
+mkdir build && cd build
+cmake \
```

Comment:
Add those for better cross compiling behaviour
```suggestion
cmake ${CMAKE_ARGS} \
```

Reply:
Thanks. Suggestions implemented.